README.md

```diff
@@ -361,6 +399,11 @@ However, there are two main reasons you might prefer True Myth to Folktale:
 
 - [ ] TODO: kind of the same as Folktale, but add specific details
 
+## Migrating from existing libs
```

Comment:
You read my mind; I wanted to add this section myself!

Reply:
:awthanks: